I'm sorry, but an uncaught exception occurred.

While running game code:
Exception: Sayer 'Juan' is not defined.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/prologue.rpyc", line 369, in script
    &����ʒl��P�U�~�j�$0?@A2*�l������$�$���9�U�q�q�P���Gh��v�a�o���M�����D��iA¾�Ė���P�CT(
  File "C:\Users\Albert Jr\Desktop\Work\renpy\renpy\ast.py", line 678, in execute
    who = eval_who(self.who, self.who_fast)
  File "C:\Users\Albert Jr\Desktop\Work\renpy\renpy\ast.py", line 576, in eval_who
    raise Exception("Sayer '%s' is not defined." % who.encode("utf-8"))
Exception: Sayer 'Juan' is not defined.

Windows-8-6.2.9200
Ren'Py 7.3.5.606
Juan Choice One Life 1.0
Tue Feb 11 13:55:31 2020
